templates/.github/workflows/js-test-and-release.yml

```diff
@@ -0,0 +1,125 @@
+name: test & maybe release
+on:
+  push:
+    branches:
+      - $default-branch # with #262 - ${{{ github.default_branch }}}
+  pull_request:
+    branches:
+      - $default-branch # with #262 - ${{{ github.default_branch }}}
+
+jobs:
+
+  check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v2
+      with:
+        node-version: lts/*
+    - uses: ipfs/aegir/actions/cache-node-modules@master
+    - run: npm run --if-present lint
+    - run: npm run --if-present dep-check
+
+  test-node:
+    needs: check
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest, ubuntu-latest, macos-latest]
+        node: [16]
+      fail-fast: true
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node }}
+      - uses: ipfs/aegir/actions/cache-node-modules@master
+      - run: npm run --if-present test:node
+      - uses: codecov/codecov-action@f32b3a3741e1053eb607407145bc9619351dc93b # v2.1.0
```

Comment:
```suggestion
      - uses: codecov/codecov-action@f32b3a3741e1053eb607407145bc9619351dc93b # v2.1.0
        with:
          directory: ./.nyc_output
          flags: node
```

Reply:
`directory` is necessary because `v2` doesn't look in the same places as `v1`.  Also `flags` give us a nice per-platform breakdown of coverage info.

Currently only node and chrome (and electron main I guess) support coverage but I've added this everywhere in case that changes in the future.